src/SeleniumLibrary/keywords/browsermanagement.py

```diff
@@ -322,6 +322,16 @@ def select_frame(self, locator):
 
         Key attributes for frames are `id` and `name.` See `introduction` for
         details about locating elements.
+        
+        See `Unselect Frame` to cancel the frame selection and return to the Main frame.
+        
+        Please note that the frame search always start from the document root or main frame.
+        
+        Example:
+        | Select Frame   | xpath=`/``/`frame[@name='top]`/`iframe[@name='left'] | # Selects the 'left' iframe |
```

Comment:
The `xpath=//...` syntax is problematic from the documentation point of view. Also we like to promote the new separator `:` in locators. It could be good idea to write the xpath like this `xpath: //...`